backend/index.js

```diff
@@ -7,10 +7,13 @@ const { StatusCodes } = require('http-status-codes');
 const { wait } = require('./lib/utils');
 const config = require('./backend.config');
 const Status = require('./services/status');
+const { getEnabledCrawlerNames } = require('./helpers/crawlerHelper');
 
 const app = express();
 const logger = pino();
-const status = new Status(config.crawlers.map((crawler) => crawler.name));
+
+const crawlersNames = getEnabledCrawlerNames(config.crawlers);
```

Comment:
This filter is used in other places. Please, update them.

Reply:
Updated